server/session/handler_item_stack_request.go

```diff
@@ -105,6 +108,11 @@ func (h *ItemStackRequestHandler) handleTake(a *protocol.TakeStackRequestAction,
 
 // handlePlace handles a Place stack request action.
 func (h *ItemStackRequestHandler) handlePlace(a *protocol.PlaceStackRequestAction, s *Session) error {
+	// Fix the container IDs before updating anything, otherwise we could have issues with future requests.
+	// These issues only happen on place, hence why we're only checking for wrong container IDs here.
+	a.Source.ContainerID = fixID(a.Source.ContainerID)
+	a.Destination.ContainerID = fixID(a.Destination.ContainerID)
```

Comment:
No point in this it seems as it's handled in the `invByID` function. The container IDs aren't wrong or broken.

Reply:
Removed.